GOVERNANCE.md

```diff
@@ -0,0 +1,135 @@
+### io.js Build Team Working Group
+
+The io.js build team is jointly governed by a Working Group (WG)
+which is responsible for high-level guidance of the project.
+
+The WG has final authority over this project including:
+
+* Technical direction
+* Project governance and process (including this policy)
+* Contribution policy
+* GitHub repository hosting
+* Conduct guidelines
+* Maintaining the list of additional Collaborators
+
+For the current list of WG members, see the project
+[README.md](./README.md#current-project-team-members).
+
+### Collaborators
+
+The [iojs/build](https://github.com/iojs/build),
+[iojs/docker-iojs](https://github.com/iojs/docker-iojs), and the
```

Comment:
If I understand correctly, it would be a working group that is only subject to the Code of Conduct of the project. It can opt to have it's own governing model or to not have a governing model at all.

Would like to get @rvagg's thoughts on this.